deps/rabbit/priv/schema/rabbit.schema

```diff
@@ -2487,6 +2487,28 @@ end}.
 {mapping, "quorum_queue.property_equivalence.relaxed_checks_on_redeclaration", "rabbit.quorum_relaxed_checks_on_redeclaration", [
     {datatype, {enum, [true, false]}}]}.
 
+
+%%
+%% Queue membership evaluation
+%%
+
+{mapping, "queue_membership_evaluation.enabled", "rabbit.member_eval_enabled", [
```

Comment:
On reflection this setting should probably be quorum queue specific so `quorum_queue_*`

Reply:
OK, will add. I assume we'd add something similar for streams in the future, but perhaps either use a separate process for it, or just separate config.